app/shared/components/Button.svelte

```diff
@@ -1,17 +1,46 @@
 <script type="ts">
   export let type = "primary";
-  export let flat = false;
+  export let ButtonOrder = "Button";
```

Comment:
Um hier eine Typ-Sicherheit zu gewährleisten würde ich hier einen neuen Typ definieren:

```
type ButtonOrder = "ValueA" | "ValueB"
```

Dann kann in der variable nur ValueA oder ValueB übergeben werden.

Im allgemeinen sollten variablen immer klein geschrieben werden.